lib/phenotype/version.rb

```diff
@@ -1,3 +1,3 @@
 module Phenotype
-  VERSION = '0.2.0'
+  VERSION = '0.3.0.beta1'.freeze
```

Comment:
so that final will get `0.3.0` just in case if we see any issues and need to create few more beta versions.